src/component.ts

```diff
@@ -1,5 +1,28 @@
-import {Component, ProviderMap} from '@loopback/core';
+import {Component, CoreBindings, ProviderMap, inject} from '@loopback/core';
+import {Class, Model} from '@loopback/repository';
+import {AuditLog} from './models';
+import {AuditLog as TenantAuditLog} from './models/tenant-support';
+import {
+  IDisableTenantGuard,
+  TenantUtilitiesBindings,
+  TenantUtilitiesComponent,
+} from '@sourceloop/core';
+import {RestApplication} from '@loopback/rest';
 
 export class AuditLogComponent implements Component {
+  constructor(
+    @inject(CoreBindings.APPLICATION_INSTANCE)
+    private readonly application: RestApplication,
+    @inject(TenantUtilitiesBindings.DisableTenantGuard, {optional: true})
+    private readonly disableTenantGuard?: IDisableTenantGuard,
```

Comment:
What is this ? Its wrong.

Reply:
this is config to not to use multi-tenancy,
this disable the tenant guard applied on repositories

Reply:
Why the name Disable tenant guard ? Why cant we take this in a config ?